test_helpers/createReq.ts

```diff
@@ -1,14 +1,10 @@
-export const createReq = (
```

Comment:
Type-annotating the arrow function would have added an additional layer of complexity to an already very difficult to read piece of code. Using the `function` keyword both makes the type-annotation easier to author, and makes the code easier to read.

Reply:
it just pollhtes diffing IMO. nothing wrong with arrow functions. its actually better to use them so you don't bind to `this` unintentionally

Reply:
Equally, if no-one makes readability changes, the code will become a mess. 
It just so happens we disagree on this one, and that's OK.
But please don't say 'never make changes like this'. Often, they are welcome.

Reply:
@Lordfirespeed I didn't say "never make changes like this", I said pushing unrelated opinionated changes to a pull request isn't good.

Reply:
The changes were relevant, because I was editing that file anyway.
I disagree with the 'opinionated' part. If you say a blanket 'no' to all differing opinions, you miss out on different, potentially better ways of doing things.